cdk-deployment/lambda/Dockerfile

```diff
@@ -6,10 +6,10 @@ COPY . /code/
 
 RUN mkdir -p /asset && \
 	python -m pip install --upgrade pip && \
-	pip install mangum uvicorn fastapi[all] -t /asset
+	pip install stac-validator mangum uvicorn fastapi[all] -t /asset --no-binary pydantic
```

Comment:
If we install stac-validator this way from pypi then we won't be able to test a new deployment on aws before we release it to pypi. It may not be a problem when we are doing minor changes but if we are doing major changes it could be I think.

Reply:
If we make changes and the validator isn't working on aws then we would have to keep releasing new versions to pypi until everything was working.